hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/DecommissionStatusSubCommand.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hdds.scm.cli.datanode;
+
+import com.google.common.base.Strings;
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.protocol.DatanodeDetails;
+import org.apache.hadoop.hdds.protocol.proto.HddsProtos;
+import org.apache.hadoop.hdds.scm.cli.ScmSubcommand;
+import org.apache.hadoop.hdds.scm.client.ScmClient;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.apache.hadoop.hdds.protocol.proto.HddsProtos.NodeOperationalState.DECOMMISSIONING;
+
+/**
+ * Handler to print decommissioning nodes status.
+ */
+@CommandLine.Command(
+    name = "decommission-status",
```

Comment:
On another look, I have concerns about adding a command that is a "continuation" of an existing command (`decomission`):

```
$ ozone admin datanode -h
Usage: ozone admin datanode [-hV] [COMMAND]
Datanode specific operations
  -h, --help      Show this help message and exit.
  -V, --version   Print version information and exit.
Commands:
  list                 List info of datanodes
  decommission         Decommission a datanode
  maintenance          Put a datanode into Maintenance Mode
  recommission         Return a datanode to service
  usageinfo            List usage information (such as Capacity, SCMUsed,
                         Remaining) of a datanode by IP address or UUID
  decommission-status  Show decommission status for datanodes
```

Some ideas:

* rename to `status` -- if it can be extended later to show other type of status info about datanodes
* rename to `status` and make it a subcommand of `decommission`; so the complete command would be `ozone admin datanode decommission status`

What do you think?

Reply:
As we want to show the status of decommissioning nodes only in this status command, I think we can go ahead with the 2nd idea. I'll make it a subcommand of `decommission`

Reply:
Thanks @Tejaskriya, and sorry for bringing it up this late.

Reply:
One more idea: instead of `decommission status`, how about `status decommission`?  To avoid accidentally starting decommissioning of a node by an incomplete command...

Reply:
Do we make it `status-decommission` as subcommand of `ozone admin datanode` instead?

Reply:
I prefer `status decommission`, because it is extensible.  While this command may be specific to decommissioning status, maybe in the future we'd like to add `status replication`, `status volumes`, or similar.

BTW, each subcommand can have aliases, so we could provide abbreviation as `ozone admin dn st decom`, if we want to save some typing (similar to `git st`, which is a commonly used alias for `git status`).  We can also do that with the single `status-decommission` subcommand, but somehow `st-decom` feels more arbitrary.

Reply:
Thanks, I'm happy with the latest version.